tests/frontend/test_kibana_app/cypress/integration/features/management/settings/miscellaneous.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Run health checks
+
+  As a kibana user
+  I want to run the health check
+  in order to validate everything is connected
+
+  Scenario: Run health check
+    Given The kibana admin user is logged in using basic authentication
+    When The user navigates to Miscellaneous settings
+    And The user runs the health checks
+    Then The health checks are displayed without errors
```

Comment:
It could be a little confusing, the goal is to check that the button works (as section actions) or that the health checks runs ok?
I think we should set the goal as just the page navigates correctly to health-check, and keep the tests for the health-check in a separate feature with a more complete test cases

Reply:
done